content/de/docs/tasks/tools/install-kubectl.md

```diff
@@ -35,7 +35,7 @@ echo "deb https://apt.kubernetes.io/ kubernetes-xenial main" | sudo tee -a /etc/
 sudo apt-get update
 sudo apt-get install -y kubectl
 {{< /tab >}}
-{{< tab name="CentOS, RHEL oder Fedora" codelang="bash" >}}cat <<EOF > /etc/yum.repos.d/kubernetes.repo
+{{< tab name="CentOS, RHEL oder Fedora" codelang="bash" >}}sudo cat <<EOF > /etc/yum.repos.d/kubernetes.repo
```

Comment:
I see this is solved in the English docs using `cat <<EOF | sudo tee /etc/yum.repos.d/kubernetes.repo`

Is there a difference ?

Reply:
I will close this PR, I saw that this file is not in the en version